npm2deb/scripts.py

```diff
@@ -56,6 +56,11 @@ def main(argv=None):
         help='license used for debian files [default: the same of upstream]')
     parser_create.add_argument(
         'node_module', help='node module available via npm')
+    parser_create.add_argument(
+                        '--no-registry', 
+                        action="store_true", 
+                        default=False, 
+                        help='Use a local tarball instead of downloading one with NPM.')
```

Comment:
Is this a boolean argument? Do you think if instead we can make it take string as an input, we can directly unzip stuff?
For example

`npm2deb create <mod-name> --no-registry <tarball-name>`

Reply:
It is, yes, but I don't think that it's worth it to implement that.  As long as the tarball fits the name-version specification, it matches what we'd get from the npm registry, which allows me to keep most of the logic from the rest of the system intact.  If I remember correctly, letting arbitrary tarballs be used just makes things harder, since then we don't have version information untill later in the process.

Since this is a rarely used option, and the tarball will be in the name-version format most of the time anyways, I didn't think it'd be worth it to try and invest the time for specifying tarballs like that.  Its pretty easy to just move the tarball to the right format.